float32 angle
float32 curLapTime
float32 damage
float32 distFromStart
float32 distRaced
float32[5] focus
float32 fuel
int32   gear
float32 lastLapTime
float32[36] opponents
int32   racePos
int32   rpm
float32 speedTH
float32 speedX
float32 speedY
float32 speedZ
float32[19] track
float32 trackPos
float32[4] wheelSpinVel
float32 z
int32 WARMUP_STAGE=0
int32 QUALIFYING_STAGE=1
int32 RACE_STAGE=2
int32 UNKNOWN_STAGE=3
int32 stage
